docs/reference/pip_install.rst

```diff
@@ -791,6 +791,7 @@ Examples
     ::
 
       $ pip install SomePackage[PDF]
+      $ pip install git+https://git.repo/SomePackage#egg=SomePackage[PDF]
```

Comment:
Sorr for nitpicking but for consistency with examples in the preceding bullet (no 5) it should be
`$ pip install git+https://git.repo/some_pkg.git#egg=SomePackage[PDF]`